src/components/BenefitPlanTabPanel.js

```diff
@@ -59,6 +60,11 @@ function BenefitPlanTabPanel({
             />
           </div>
           <div style={{ float: 'right', paddingRight: '16px' }}>
+            <Contributions
```

Comment:
Personally I'd use PublishedComponent since you you want to get a particular component. However it also works with contribution.

Reply:
yes, let's keep contribution, in that file contribution is used in another places hence I don't want to combine those two method.